addons/api/tests/unit/handlers/indexed-db-handler-test.js

```diff
@@ -147,6 +148,52 @@ module('Unit | Handler | indexed-db-handler', function (hooks) {
         'server handler should be called once for each page',
       );
     });
+
+    test('it loads from indexeddb only and does not fetch from api, when peekIndexedDB option is used', async function (assert) {
```

Comment:
This test is chefkiss for real. Very clever to test the process I will take some of the approaches here for the future sorting tests 😃